### PR TITLE
feat(janitor): wash mop

### PR DESCRIPTION
**Pull request by Prakkkmak**

### CODE REVIEW

**Comment by Prakkkmak** [comment]
To be tested =>

File: code/Modules/Items/Carriables/ItemCleaner.cs [hunk 1]
```diff
@@ -16,13 +19,41 @@ public ItemCleaner(ItemCleanerData itemData) : base(itemData)
...
+        /// <param name="bucket"></param>
+        private void Wash(Bucket bucket)
+        {
+            bucket.WasteWater(Dirtyness);
+            Dirtyness = 0;
+        }
+        
+        /**
+         * Clean stain with the cleaning value. The cleaning have to be decreased by the current dirtyness.
+         */
```
Comment (Nowp) [inline code/Modules/Items/Carriables/ItemCleaner.cs:52]: Not javadoc

File: code/Modules/Items/Carriables/ItemCleaner.cs [hunk 2]
```diff
@@ -16,13 +19,41 @@ public ItemCleaner(ItemCleanerData itemData) : base(itemData)
...
+        private void Clean(Stain stain)
+        {
+            stain.CleanStain(Data.CleaningValue);
+            Dirtyness++;
```
Comment (Nowp) [inline code/Modules/Items/Carriables/ItemCleaner.cs:56]: Dirtyness += DirtFactor; ?
We could make the mop getting dirty quicker

**Review by Nowp — COMMENTED** [review]
also magic numbers

File: code/Modules/Items/Carriables/ItemCleaner.cs (hunk 1 above)
Comment (Prakkkmak) [reply code/Modules/Items/Carriables/ItemCleaner.cs:52]: fixed

File: code/Modules/Items/Carriables/ItemCleaner.cs (hunk 2 above)
Comment (Prakkkmak) [reply code/Modules/Items/Carriables/ItemCleaner.cs:56]: I want to keep it simple for now

Comment (Nowp) [reply code/Modules/Items/Carriables/ItemCleaner.cs:56]: And there's no maximum for dirtyness '/?

**Pull request merged**